libs/core/package.json

```diff
@@ -9,8 +9,14 @@
     "type": "git",
     "url": "https://github.com/orchestratora/orchestrator.git"
   },
-  "keywords": [],
-  "author": "",
+  "keywords": [
+    "rendering",
+    "extensions",
+    "orchestrator",
+    "components",
+    "angular"
+  ],
+  "author": "Orchestrator <orchestratora@gmail.com>",
```

Comment:
Wouldn't it be better to have a domain for orchestrator?

Reply:
yeah, it might be good, but we do not have now.
Can be something like orchestrator.org (available)?

Reply:
orchestrator.io :P

Reply:
It's already taken :(

Reply:
how about orchestrator.rocks ? =)